rtrlib/rtr/rtr.h

```diff
@@ -39,6 +48,31 @@ enum rtr_rtvals {
     RTR_INVALID_PARAM = -2
 };
 
+enum interval_range {
+    BELOW_INTERVAL_RANGE = -1,
+    INSIDE_INTERVAL_RANGE = 0,
+    ABOVE_INTERVAL_RANGE = 1
+};
+
+enum interval_type {
+    EXPIRATION = 0,
```

Comment:
explicite values are not needed here, enums start with `0` by default and increment by `1`, too.